client/component-lab/Notification/Notification.stylus

```diff
@@ -0,0 +1,184 @@
+@import '~lab/Colors/Colors.stylus'
```

Comment:
He needs to keep it this way, because `stylus` extension is required for local scoped css.